api/src/main/java/org/pmiops/workbench/db/dao/WgsExtractCromwellSubmissionDao.java

```diff
@@ -1,11 +1,17 @@
 package org.pmiops.workbench.db.dao;
 
 import java.util.List;
+import java.util.Optional;
 import org.pmiops.workbench.db.model.DbWgsExtractCromwellSubmission;
 import org.pmiops.workbench.db.model.DbWorkspace;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.query.Param;
 
 public interface WgsExtractCromwellSubmissionDao
     extends CrudRepository<DbWgsExtractCromwellSubmission, Long> {
   List<DbWgsExtractCromwellSubmission> findAllByWorkspace(DbWorkspace dbWorkspace);
+
+  @Query(value = "SELECT * FROM wgs_extract_cromwell_submission WHERE wgs_extract_cromwell_submission_id = :workspaceId and workspace_id = :jobId", nativeQuery = true)
```

Comment:
using `@Query` here because otherwise I'd have to make the function name `findByWorkspaceWorkspaceIdAndWgsExtractCromwellSubmissionId` which I believe is a violation of several international treaties

Reply:
It's ugly... but I think I'd probably just use the autogenerated function name. Isn't there an extra `workspace` in there or am I missing something? This is less error prone (case in point: currently written query has a bug with placement of the parameters)

Also - why didn't the tests break from the param order mixup? I guess because of native query (or are the tests overly aggressive about mocking?)

Reply:
argh. Probably tests are overly aggressive about mocking. Can fix when I fix this.

There is not an extra `workspace` in there. The property on `DbWorkspace` the query is looking for is `workspaceId`.

Reply:
The issue with the test was the IDs were both `1`.